packages/atomic/cypress/integration/result-list/result-components/result-multi-value-text.cypress.ts

```diff
@@ -91,24 +99,29 @@ describe('Result MultiValueText Component', () => {
           .init();
       });
 
-      assertRemovesComponent(ResultMultiValueTextSelectors.firstInResult);
+      assertRemovesComponent(() =>
+        ResultListSelectors.firstResult().find(resultMultiValueTextComponent)
+      );
     });
 
     describe('when the field value is not a string nor a string array', () => {
       beforeEach(() => {
         new TestFixture()
           .with(addMultiValueText({field: 'hello'}))
-          .withCustomResponse((response) => {
-            response.results.forEach((result) => (result.raw['hello'] = 420));
-          })
+          .with(addFieldValueInResponse('hello', 420))
           .init();
       });
 
-      assertRemovesComponent(ResultMultiValueTextSelectors.firstInResult);
+      assertRemovesComponent(() =>
+        ResultListSelectors.firstResult().find(resultMultiValueTextComponent)
+      );
       assertConsoleError();
     });
 
-    describe('when the field value exists & is a string array', () => {
+    function testWithValidFieldValue(
+      getFieldValues: (valuesToTest: string[]) => string | string[],
+      delimiter?: string
+    ) {
```

Comment:
I'm repeating this whole series of test twice. Once with a string array, and once with a single string and a delimiter.

I want to ensure both are treated equally.